PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -5558,12 +5558,29 @@ void Progress::close()
 }
 
 
-void Progress::setPercent(unsigned percent, const TCHAR *fileName) const
+void Progress::setPercent(unsigned percent, const TCHAR* fileName, int nbHitsSoFar) const
 {
 	if (_hwnd)
 	{
 		::PostMessage(_hPBar, PBM_SETPOS, percent, 0);
 		::SendMessage(_hPText, WM_SETTEXT, 0, reinterpret_cast<LPARAM>(fileName));
+		const int stringSize = 32;
+		TCHAR str[stringSize];
+		swprintf(str, stringSize, TEXT("%d hits"), nbHitsSoFar);
+		::SendMessage(_hPRunningHitsText, WM_SETTEXT, 0, reinterpret_cast<LPARAM>(str));
+	}
+}
+
+
+void Progress::setInfo(const TCHAR* info, int nbHitsSoFar) const
+{
+	if (_hwnd)
+	{
+		::SendMessage(_hPText, WM_SETTEXT, 0, reinterpret_cast<LPARAM>(info));
+		const int stringSize = 32;
+		TCHAR str[stringSize];
+		swprintf(str, stringSize, TEXT("%d hits"), nbHitsSoFar);
+		::SendMessage(_hPRunningHitsText, WM_SETTEXT, 0, reinterpret_cast<LPARAM>(str));

```

Comment:
When progress is updated, either `setPercent()` is called, or `setInfo()` is called.  setPercent is called when the progress-bar is involved in the update, otherwise setInfo is called.  Since both the filename and the new hit count need to be updated in both cases, there is a `SendMessage(_hPRunningHitsText, ...)` call in both places.